recipes/slang/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "slang" %}
+{% set version = "2.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://www.jedsoft.org/releases/slang/slang-{{ version }}.tar.bz2
+  sha256: fc9e3b0fc4f67c3c1f6d43c90c16a5c42d117b8e28457c5b46831b8b5d3ae31a
+
+build:
+  number: 0
+  run_exports:
+    # This package has been extremely stable
+    # https://abi-laboratory.pro/index.php?view=timeline&l=slang
+    - {{ pin_subpackage(name, max_pin='x') }}
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - libiconv
+    - libpng
+    - oniguruma 6.9.*
+    - pcre
+    - readline
+    - zlib
+  run:
+    - oniguruma 6.9.*
+
+test:
+  commands:
+    - slsh --version
+
+about:
+  home: http://www.jedsoft.org/slang
+  license: GPL
```

Comment:
Could you please add the version of GPL in the feedstock?

Reply:
No problem, that is taken care in conda-forge/slang-feedstock#2.